services/positions-api/src/positions/post.rs

```diff
@@ -35,12 +35,12 @@ impl Service for PositionsPostHandler {
     type Error = hyper::Error;
     type Future = FutureHandled;
 
-    fn call(&self, req: Request) -> Self::Future {
+    fn call(&self, mut req: Request) -> Self::Future {
```

Comment:
Why was mutability needed here?

Reply:
BEcause remove() is a mutating operation